script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -523,8 +523,8 @@ Because if I don't, then everything―――
 -- Page 336, Offset 32595.
 -- しかし、いかに神秘を学ぼうと、どれほど長寿の生物がいようと、無に還る<終|つい>―――本質的な死を体験する事はできない」
 And yet,
- even if someone were to devote themselves to studying the Arcane,
- even if someone were to have a lifespan that stretched on for eons,
+ no matter how much someone would study the Arcane,
+ no matter how long someone's lifespan was to become,
```

Comment:
Original reads more fluently to me here.

Reply:
It kind of does, but at the same time it also completely overcomplicates the sentence while at the same time ignoring the structure of the Japanese about the "extent" of things. These kinds of "No matter how hard you try", etc sentences aren't uncommon in English at all, so I don't see the reason why you'd revert to just "Even if you did x", it's a little odd.

Reply:
Not sure I agree with 'overcomplicates'. I also don't really think this elides the どれほど, I feel the same idea is still encoded. Still, `no matter how much` is a fine formulation, it's the second half that doesn't sit right. "no matter how much someone _would study_" feels slightly confused, similarly "was to become" for lifespan feels like it could be more naturally expressed. Yusuke's comment seems ok to me.

Reply:
I concur. 
I think Yusuke's version has the same issue--"no matter how much one were to study the arcane" doesn't really parse to my eyes.

Reply:
> And yet,
>  no matter how much you study the Arcane,
>  no matter how long your lifespan becomes,
>  You will never be able to experience that return to nothingness―――true death."

What about this?

Reply:
Yeah, that works for me